manager/base.py

```diff
@@ -134,16 +134,18 @@ def auth_common(identity, x_rh_identity):  # pylint: disable=unused-argument
 
     if 'account_number' not in id_details:
```

Comment:
`'account_number' not in in_details and 'org_id' not in id_details` just for correctness

Reply:
fixed